src/app/featureFlags/constants.ts

```diff
@@ -0,0 +1,14 @@
+import { FeatureFlagVariantValue } from './types';
+
+export const experiments: Record<string, FeatureFlagVariantValue[]> = {
+  searchButton: [
+    'originalButton',
+    'bannerColorButton',
+    'grayButton',
+  ],
+};
+
+export const experimentIds: { [key: string]: keyof typeof experiments } = {
+  '473Jeu7ORta1TTuJ5UjL6w': 'searchButton', // prod
```

Comment:
this test experiement had already been started, here is a new one i just made that is in draft state and we'll start it after the release goes out `zHtr33wcS1mVgFn7b2nU3A`